frontend/src/scenes/insights/InsightTooltip/InsightTooltip.scss

```diff
@@ -0,0 +1,89 @@
+@import '~/vars';
+
+.ph-graph-tooltip {
+    padding: 0 !important;
+    overflow: hidden;
+    border: 1px solid $border;
+    box-shadow: 0 4px 12px rgba(0, 0, 0, 0.1);
+    font-size: 14px;
+    z-index: $z_graph_tooltip;
+    transition: all 0.4s;
+    max-width: 480px;
+    border-radius: $radius;
+    background: #fafafa;
+
+    .LemonTable {
+        font-size: 0.75rem;
+        border: none;
+        border-bottom-left-radius: 0;
+        border-bottom-right-radius: 0;
+        border-bottom: 1px solid $border;
+
+        .color-column {
+            padding: 0;
+            width: 6px;
+        }
+
+        .datum-column {
+            padding-left: 0.5rem;
+        }
+
+        .LemonTable__scroll {
+            width: 100%;
+            overflow: hidden;
+        }
+        .LemonTable__content > table > thead {
+            text-transform: none;
+
+            .datum-column {
+                .LemonTable__header-content {
+                    white-space: nowrap;
+                }
+            }
+            .series-column-header {
+                margin-left: $default_spacing / 4;
+            }
+        }
+
+        .LemonTable__content > table > tbody {
+            .color-column {
+                position: relative;
+
+                .color-cell {
+                    position: absolute;
+                    top: 0;
+                    bottom: 0;
+                    width: 6px;
+                }
+            }
+            .datum-label-column {
+                font-weight: 600;
+            }
+            .series-data-cell {
+                font-weight: 600;
+            }
+            .tag-pill {
+                background-color: rgba(0, 0, 0, 0.08);
+                margin-right: 0;
+                border: 0;
+                color: $bg_navy;
+                overflow: hidden;
+                vertical-align: middle;
+            }
+        }
+    }
+
+    .table-subtext {
+        font-size: 0.75rem;
+        text-align: center;
+        padding: $default_spacing / 2 $default_spacing;
+        color: $text_muted;
+        background-color: $bg_light;
+
+        .table-subtext-click-to-inspect {
+            margin-top: $default_spacing / 4;
+            color: $primary;
```

Comment:
The blue text color of "Click to view users" makes me think it's a clickable link, while it's just a suggestion for the user. Slightly frustrating because it makes the tooltip look interactive when it isn't at all. I propose using gray similar to what the previous tooltip iteration.
<img width="196" alt="Screen Shot 2022-01-06 at 12 28 07" src="https://user-images.githubusercontent.com/4550621/148376336-c82dd790-4557-4557-87e0-ec279ff80320.png">